debian/rules

```diff
@@ -1,7 +1,13 @@
 #!/usr/bin/make -f
 export PYBUILD_NAME=dangerzone
 export DEB_BUILD_OPTIONS=nocheck
+#export PYBUILD_INSTALL_ARGS=--install-lib=/usr/lib/python3/dist-packages
+#export PYTHONDONTWRITEBYTECODE=1
+#export DH_VERBOSE=1
```

Comment:
Yikes, I just realized these options are disabled. I think they should be enabled actually, but I'll make a cross-check with #748, once it works again. In the meantime, I've enabled them in 3892dec4.